examples/todoapp/usecases/deleteItem.js

```diff
@@ -0,0 +1,45 @@
+const { Ok, Err, usecase, step, ifElse } = require('../../../src/buchu')
```

Comment:
Is this file part of the solution?

Reply:
not really, I just thought about implementing a test Use Case to give an example using the new feature, was too much?